### ISSUE TITLE
an h3 inside a caption is not read by screen reader as an h3

### ISSUE THREAD

**Issue by Techlord007** [nvaccess/nvda#17169]
Steps to reproduce:
Open attached PDF file. 
Use the down arrow to go through the document till you get to page two.
When the reader gets to the words "Contracts awarded" (which is a complete h2), the h2 expands to include the <p tag and the caption with an h3 inside it.

Actual behavior:
The h3 inside the table caption is read as part of the h2

Expected behavior:
The h3 inside the caption should be read as an h3 header.

NVDA logs, crash dumps and other attachments:
There was no crash

System configuration
NVDA installed/portable/running from source:
Dell laptop

NVDA version:
Cannot find a way to determine this on a PC.
 
Windows version:
Microsoft windows 10 Enterprise

Name and version of other software in use when reproducing the issue:
Fails in Chrome, Edge. Works great in Firefox.

Other information about your system:
Processor	Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz   1.80 GHz
Installed RAM	16.0 GB (15.9 GB usable)
Product ID	00329-00000-00003-AA666
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

Other questions
Does the issue still occur after restarting your computer?
Yes

Have you tried any other versions of NVDA? If so, please report their behaviors.
No

If NVDA add-ons are disabled, is your problem still occurring?
I have added nothing to it since installation

Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
I don't see any menu at all when NVDA starts. I just get that panel that shows what text is being spoken
[supplier-history.pdf](https://github.com/user-attachments/files/16997755/supplier-history.pdf)

**Comment by seanbudd** [comment]
Thanks for reporting this.
Can you please report this to chromium, as it is likely a bug there

**Comment by Adriani90** [comment]
@Techlord007 did you report this to Chromium? Can you please post a link to the bug report so we can track it here as well?
Are you having this issue still with NvDA 2025.1 Beta in latest Chromium browser updates?
